docs/book/src/tasks/experimental-features/cluster-class/upgrade-cluster.md

```diff
@@ -21,6 +21,7 @@ Change `1.21.2` to `1.22.0` as below.
 ```bash
 kubectl patch cluster clusterclass-quickstart --type json --patch '[{"op": "replace", "path": "/spec/topology/version", "value": "v1.22.0"}]'
 ```
+**Important Note**: A +2 minor Kubernetes version upgrade is not allowed in Cluster Topologies. Example: Upgrading from `1.21.2` to `1.23.0` is no allowed.
```

Comment:
```suggestion
**Important Note**: A +2 minor Kubernetes version upgrade is not allowed in Cluster Topologies. Example: Upgrading from `1.21.2` to `1.23.0` is not allowed.
```
nit

Should we slightly mention that this limitation also exists with plain CAPI/KCP?

Otherwise it comes across like CC is less powerful in this regard.

Reply:
>Should we slightly mention that this limitation also exists with plain CAPI/KCP?
>Otherwise it comes across like CC is less powerful in this regard.

Sounds good. I think it would be better to mention that.